packages/components/src/components/link/component.tsx

```diff
@@ -90,7 +90,7 @@ export class KolLinkWc implements API {
 			href: typeof this.state._href === 'string' && this.state._href.length > 0 ? this.state._href : 'javascript:void(0);',
 			target: typeof this.state._target === 'string' && this.state._target.length > 0 ? this.state._target : undefined,
 			rel: isExternal ? 'noopener' : undefined,
-			download: (typeof this.state._download === 'string' && this.state._download === 'true') || this.state._download === true ? true : undefined,
+			download: this.state._download === true || this.state._download === 'true' ? '' :  this.state._download,
```

Comment:
Hi @cyrdam 

Entweder `true` oder `string` oder `undefined`, nicht `false` oder sonstiges.

Kannst Du das bitte so umformulieren.

Danke

Reply:
Wenn wir den Wert `'true'` mitgeben, dann wird dies auch ausgegeben: `download="true"`, anstatt nur `download`.

Ich schreibe es später um, gehe davon aber aus, dass es mehr Code generiert.

Reply:
Falls `"true"` und `"false"` keine Umwandlung zu einem Boolean brauchen, dann würde ich die Logik in folgendes umwandeln:

```typescript
download: typeof this.state._download === 'string' ? this.state._download : this.state._download === true ? '' : undefined,
```

`undefined` und `false` wären damit ausgeschlossen, `"undefined"`, `"true"` und `"false"` wären in der Ausgabe enthalten. Damit müsste jedes Projekt vorher eine Abfrage machen, ob die Strings Booleans sind/beinhalten.